### ISSUE TITLE
Support acala wallet 

### ISSUE THREAD

**Issue by youyou0625** [jacogr/sample-ledger-ed25519#13]
Hello, 
I want to use for an acala wallet, this is possible?
Thanks for help

**Comment by clock-workorange** [comment]
yes, you need to edit the index.js and manually use the BIP44 app code in this case acala is 787, polkadot 354 and astar 810
